src/FlaUI.WebDriver.UITests/ElementTests.cs

```diff
@@ -175,6 +175,51 @@ public void SendKeys_ShiftedCharacter_ShiftIsReleased()
             Assert.That(element.Text, Is.EqualTo("!1Test TextBox"));
         }
 
+        [Test]
+        public void SendKeys_Qwerty()
```

Comment:
The test method names should follow the pattern `command - state under test - expected behavior`.
```suggestion
        public void SendKeys_UpperAndLowercaseLetters_EntersIdenticalText()
```